src/main/java/Sysc4806Group/demo/controllers/BookController.java

```diff
@@ -94,4 +92,11 @@ public String updateBook(@ModelAttribute Book book, Model model) {
         repository.save(bookFromDB);
         return "view-book";
     }
+
+    @GetMapping("/addItem/{id}")
+    public String addItemForm(@PathVariable String id, Model model) throws Exception {
```

Comment:
I forgot to remove this.....pls remove this

Reply:
Hey Dhyan, 
I deleted those lines! Gotchu!